scalikejdbc-core/src/main/scala/scalikejdbc/interpolation/SQLSyntax.scala

```diff
@@ -255,7 +255,8 @@ object SQLSyntax {
     }
     apply(value, parameters)
   }
-  def csv(parts: SQLSyntax*): SQLSyntax = join(parts, sqls",", false)
+  def csv(parts: SQLSyntax*): SQLSyntax = commaSeparated(parts)
+  private[scalikejdbc] def commaSeparated(parts: scala.collection.Seq[SQLSyntax]): SQLSyntax = join(parts, sqls",", false)
```

Comment:
I understand we have to have another name for this. I prefer `toCSV` than `commaSeparated`.

Reply:
renamed and `push -f`